src/Bookings.js

```diff
@@ -19,6 +35,8 @@ const Bookings = () => {
       <div className="container">
         <Search search={search} />
         <SearchResults bookings={bookings} />
+        <FormBooking handleAddNewBooking={handleAddNewBooking} />
```

Comment:
Passing functions around to lift up state, is a very common React pattern so good to see you using it to solve this issue rather than somekind of bodge